Formula/dartsim.rb

```diff
@@ -6,6 +6,7 @@ class Dartsim < Formula
   revision 1
 
   bottle do
+    rebuild 1
```

Comment:
I don't know if we're supposed to bump this. I thought I saw somewhere that we're just supposed to bump the formula revision

Reply:
Right, it seems bumping the formula revision is the right way to rebuild bottles per dependency updates (see: https://github.com/jslee02/homebrew-core/commit/5d3644c33079df05375d4b15a6ade8beccd50edf). Thanks @scpeters !